backend/src/apiserver/main.go

```diff
@@ -106,10 +107,25 @@ func main() {
 	}
 	log.SetLevel(level)
 
+	backgroundCtx, backgroundCancel := context.WithCancel(context.Background())
+	defer backgroundCancel()
```

Comment:
@hbelmiro / @mprahl looks like we are cancelling the context at the end anyway, is this second call needed?

Reply:
I think we can remove [this call](https://github.com/kubeflow/pipelines/pull/11469/files#diff-d3ae27a3b5003f91b2cbf384575c21f07dd142165daf842fc79fd8459a844829R118) instead.

Reply:
There's no harm in keeping it since canceling an already canceled context is a no-op. The "benefit" I suppose is if the HTTP server panics, the context would still cancel but the rest of the code isn't going to wait for goroutine to exit cleanly anyways. Feel free to remove this one but we need to keep the other call or else `wg.Wait()` will finish only after `reconcileSwfCrs` is done.